src/status_im/ethereum/transactions/core.cljs

```diff
@@ -13,16 +13,27 @@
 
 (def confirmations-count-threshold 12)
 
-(def etherscan-supported? #{:testnet :mainnet :rinkeby})
+(def etherscan-supported? #{1 3 4})
+(def binance-mainnet-chain-id 56)
```

Comment:
probably better to move to constants, you define it in two namespaces

Reply:
fixed